src/lib/controller/message_handler.ts

```diff
@@ -14,7 +14,7 @@ export class ControllerMessageHandler {
     switch (message.command) {
       case ControllerCommand.beginInclusion: {
         const success = await driver.controller.beginInclusion(
-          message.includeNonSecure
+          message.includeNonSecure!
```

Comment:
Are these other changes needed?

Reply:
There is an `| undefined` missing from the function signature. You know what - I should probably do a 8.1.1 anyways.